tests/e2e/README.md

```diff
@@ -0,0 +1,34 @@
+# End-to-End Testing
+
+## Context
+
+The FSx OpenZFS CSI driver utilizes a variety of workflows to validate end to end functionality in a Kubernetes environment.
+The framework and existing testsuites for Kubernetes is leveraged to test common cases.
+Untested or unique test cases are also tested using a custom testsuite.
+
+Due to the nature end-to-end tests it is possible to leave testing artifacts stranded.
+Avoid abruptly terminating the testing process to allow FSx Resources to be cleaned properly.
+
+## Running
+
+In order to run the E2E tests, an EKS cluster with the OpenZFS driver installed must be utilized.
+Follow the respective section depending on if a test environment is already set up.
+
+### Existing Environment
+
+The test environment should contain the following:
+1. Kubernetes 1.13+ (CSI 1.0).
+2. EKS Cluster
+3. Ginkgo CLI
+
+Run `ginkgo -r -p` from the tests/e2e folder to run
+
+### Empty Environment
+
+To create a testing environment the following must be preinstalled:
+1. Amazon CLI
+
+TODO
```

Comment:
Is this supposed to still be here

Reply:
Yea, the next revision is going to include this section and bash scripts associated with it. Wanted to get initial feedback on the tests themselves first.